config/typescript/webpack.common.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+const SizePlugin = require('size-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const { CheckerPlugin } = require('awesome-typescript-loader');
+
+const PATHS = require('./paths');
+
+// To re-use webpack configuration across templates,
+// CLI maintains a common webpack configuration file - `webpack.common.js`.
+// Whenever user creates an extension, CLI adds `webpack.common.js` file
+// in template's `config` folder
+const common = {
+  output: {
+    // the build folder to output bundles and assets in.
+    path: PATHS.build,
+    // the filename template for entry chunks
+    filename: '[name].js',
+  },
+  devtool: 'source-map',
+  stats: {
+    all: false,
+    errors: true,
+    builtAt: true,
+  },
+  module: {
+    rules: [
+      {
+        test: /\.ts$/,
+        use: 'awesome-typescript-loader'
+      },
+      // Help webpack in understanding CSS files imported in .js files
+      {
+        test: /\.css$/,
+        use: [MiniCssExtractPlugin.loader, 'css-loader'],
+      },
+      // Check for images imported in .js files and
+      {
+        test: /\.(png|jpe?g|gif)$/i,
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              outputPath: 'images',
+              name: '[name].[ext]',
+            },
+          },
+        ],
+      },
+    ],
+  },
+  plugins: [
+    new CheckerPlugin(),
+    // Print file sizes
+    new SizePlugin(),
+    // Copy static assets from `public` folder to `build` folder
+    new CopyWebpackPlugin([
+      {
+        from: '**/*',
+        context: 'public',
+      },
+    ]),
```

Comment:
Will not work now :(
Should be migrated to the new format:
```suggestion
    new CopyWebpackPlugin({
      patterns: [
        {
          from: '**/*',
          context: 'public',
        },
      ]
    }),
```